src/shared/components/home/signup.tsx

```diff
@@ -296,7 +325,12 @@ export class Signup extends Component<
               {this.state.registerRes.state === "loading" ? (
                 <Spinner />
               ) : (
-                this.titleName(siteView)
+                [
+                  this.titleName(siteView),
+                  ...(oauth_provider !== undefined
+                    ? [`(${oauth_provider.display_name})`]
+                    : []),
+                ].join(" ")
```

Comment:
This could be written as:
```suggestion
                [
                  this.titleName(siteView),
                  ...(oauth_provider
                    ? [`(${oauth_provider.display_name})`]
                    : []),
                ].join(" ")
```